src/main/kotlin/racingcar/domain/RacingCar.kt

```diff
@@ -2,19 +2,33 @@ package racingcar.domain
 
 import racingcar.domain.carracer.CarRacer
 
-private const val DEFAULT_MOVEMENT = 0
-private const val MOVING_BENCHMARK = 4
+class RacingCar(
+    name: Name,
+    movement: Movement,
+    private val carRacer: CarRacer,
+) {
+    private var _name: Name = name
+    private var _movement: Movement = movement
+    val name: String
+        get() = _name.value
```

Comment:
name은 바뀔일이 없어보입니다. 주 생성자에 프로퍼티로 위치해있어도 문제가 없어보이는데 제가 파악하지 못한 의도가 있을까요?
```suggestion
    val name: Name,
    movement: Movement,
    private val carRacer: CarRacer,
) {
    private var _movement: Movement = movement
```